app/routes/clients.tsx

```diff
@@ -183,6 +251,40 @@ export default function Clients({ loaderData }: Route.ComponentProps) {
           </Table>
         </Col>
       </Row>
+      {/* Pagination */}
+      {totalPages > 1 && (
+        <Row className="mb-4">
+          <Col>
+            <Pagination className="justify-content-center">
+              <Pagination.First
```

Comment:
Why not use `href` attribute of `Pagination.Item` instead of handling navigation programmatically ?